addon-test-support/backstop.js

```diff
@@ -206,10 +206,28 @@ function validateName(name) {
   }
 }
 
+/**
+ * Reads environment.js config from the meta tag in test page, and returns the config object related to this addon.
+ */
+function getAddonCfgFromParentApp() {
+  const appEnvConfig = document.querySelector('meta[name*="config/environment"]').getAttribute('content');
+  return JSON.parse(decodeURIComponent(appEnvConfig))['ember-backstop'];
```

Comment:
I think we need to guard here against environment.js files which do not have `ember-backstop` nodes.  Maybe be sure to return an object.